proxy-explanation/draft.md

```diff
@@ -72,13 +73,14 @@ The "application/proxy-explanation+json" media type denotes a JSON {{RFC7159}} f
 * **name** - A short string identifying the party operating the proxy
 * **title** - A short string title for the explanation
 * **description** - A string explaining why the request wasn't successful
-* **moreinfo** - A string containing an absolute URL {{RFC3986}} which the user can find relevant information.
 
-The "name" and "title" members MUST be present; all other members are OPTIONAL.
+The "name" and "title" members MUST be present; all other members are OPTIONAL. All strings MUST be encoded in UTF-8 {{RFC3629}}.
+
+Proxies MUST NOT include any URL in any of the fields and client SHOULD prevent the display of obfuscated URLs to the end user.
```

Comment:
As indicated on https://bugzilla.mozilla.org/show_bug.cgi?id=637619 It may be right and proper to leave this decision to browser implementations.

I would have use for this feature myself (see below). I was just trying to be safe as I could see how users could start to click on a usual link location without looking much.

For example

    Policy Violation
    
    The proxy "Exa Networks" says:
    
      This website is categorised as "Gaming" and is not accessible from users from "Year 6".
    
    For more information, see: 
      Should you believe the categorisation is inaccurate,
      want to add this category to the allowed category list, or
      want to add this site as an exception to your whitelist
      please update your school setting following
       "https://surfprotect.com/reclassify/year%206/https://www.minecraft.com"

with a MIM could become

       "http://surfprotect.com/reclassify/year%206/https://www.minecraft.com"

Which would most likely also go to the same proxy and could be attacked to steal teachers username and password. But if browser implement linking it would make my product much better :smiley: